Assets/HoloToolkit/SpectatorView/Scripts/SpatialSync/CameraCaptureHololens.cs

```diff
@@ -0,0 +1,167 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.using UnityEngine;
+
+using System.Collections.Generic;
+using UnityEngine;
+using System.Linq;
+
+#if NETFX_CORE
+using UnityEngine.XR.WSA.WebCam;
+#endif
+
+namespace HoloToolkit.Unity.SpectatorView
+{
+    /// <summary>
+    /// Manages the camera capture on the HoloLens
+    /// </summary>
+    public class CameraCaptureHololens : MonoBehaviour
+    {
+		// Executed once a frame has succesfully been captured
+        public delegate void FrameCapturesDelegate(List<byte> frameData, int width, int height);
+        public FrameCapturesDelegate OnFrameCapture;
+
+#if NETFX_CORE
+        /// <summary>
+        /// Manages the camera capture
+        /// </summary>
+		private PhotoCapture photoCaptureObject;
+
+        /// <summary>
+        /// Is the HoloLens capturing photos?
+        /// </summary>
+        private bool capturing = false;
+#endif
+
+        /// <summary>
+        /// Width of the photo taken
+        /// </summary>
+        private int photoWidth;
+
+        /// <summary>
+        /// Height of the photo taken
+        /// </summary>
+        private int photoHeight;
+
+        /// <summary>
+        /// Texture to which the photo will be saved to
+        /// </summary>
+        private Texture2D targetTexture;
+
+        /// <summary>
+        /// Starts capturing photos
+        /// </summary>
+        public void StartCapture()
+        {
+#if NETFX_CORE
+			if(!capturing)
+			{
+				PhotoCapture.CreateAsync(false, OnPhotoCaptureCreated);
+			    capturing = true;
+			}
+#else
+			Debug.LogWarning("Capturing only supported on the HoloLens platform");
+#endif
+        }
+
+        /// <summary>
+        /// Stops capturing photos
+        /// </summary>
+        public void StopCapture()
+        {
+#if NETFX_CORE
+			if(capturing)
+			{
+				photoCaptureObject.StopPhotoModeAsync(OnStoppedPhotoMode);
+			}
+			capturing = false;
+#else
+			Debug.LogWarning("Capturing only supported on the HoloLens platform");
```

Comment:
NETFX_CORE covers more than just HoloLens, but WMR and UWP apps respectively. This statement isn't entirely true.

Reply:
@StephenHodgson, do you have a suggested condition? I agree the comment isnt completely accurate

Reply:
Just something simpler like "Capturing isn't supported on this platform"